### DISCUSSION TITLE
Changes to work at 120Mhz

### DISCUSSION THREAD

**Discussion by sonik-br** [dangiu/PicoMemcard#75] [General]
Hi.
What would be the required changes to get it working at 120MHz.
I'm trying to get it working as a gamepad and also using PIO-USB that requires the clock at 120 or 240.

**Comment by dangiu** [comment]
Should be a simple call to `set_sys_clock_khz()`. Right now it's running at the default frequency of 125MHz. You can try to change it and see if everything behaves the same.
I actually am not too sure of how the function I mentioned changes the frequency under the hood... If it also messes with the PIO frequency you may need to adjust those. From what I recall those are completely independend so you should be fine :)

**Comment by sonik-br** [comment]
Thank you. I'm going to give it a try.
